src/events/MessageReactionAdd.ts

```diff
@@ -0,0 +1,16 @@
+import { logErrorFunctions } from '../translations/logs.js';
+import { getConfigProperty } from '../utils/config.js';
+import { addExperience } from '../utils/experience.js';
+import { logger } from '../utils/logger.js';
+import { type ClientEvents, Events, type MessageReaction, type User } from 'discord.js';
+
+export const name = Events.MessageReactionAdd;
+
+async function removeKromidReaction(messageReaction: MessageReaction, _user: User) {
+  if (messageReaction.message.author.id !== "206360333881704449") return;
+  if (messageReaction.emoji == "🧅") await messageReaction.remove();
```

Comment:
This will only prevent the 🧅 emoji from being added, but will not stop custom drachevski kromid emojis from being used as a bullying mechanism.

Reply:
I have added the functionality of detecting similar emojis that include the name "kromid" or "onion".